CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ if (UBSAN)
     set(CMAKE_LINKER_FLAGS "${CMAKE_LINKER_FLAGS} -fno-omit-frame-pointer -fsanitize=undefined")
 endif()
 
+set(CMAKE_CXX_FLAGS_DEBUG_INIT "-g -Og")
+set(CMAKE_CXX_FLAGS_RELEASE_INIT "-g -Og")
```

Comment:
so now the release build is not `-O3`? not good.
Maybe it should be conditioned with `if (TESTING)` ?